### DISCUSSION TITLE
Change display text from title to description

### DISCUSSION THREAD

**Discussion by MrBaloo1337** [MMRIZE/MMM-CalendarExt3#154] [General]
I wonder if there is any possibilty so show the description in a calender event instead of the title?

For example the iCal sends everyday "Sport, training" in title and in the description theres is what type of training it is.

Regards

**Comment by eouia** [comment]
```js
eventTransformer: (ev) =>{
	const targetCalendars = [ 'SpecificCalendarName', 'AntotherCalendarName' ]; // Fulfill the calendar names you want to apply.
	if (targetCalendars.includes(ev.calendarName) && ev.description) {
		ev.title = ev.description
	}
	return ev
}
```
Add this into your CX3 configuration.

**Comment by MrBaloo1337** [comment]
Awesome, works like a charm! Thanks :)
